version: "3.3"
services:
  fastapi:
    image: datascientest/fastapi:1.0.0
    container_name: fastapi
    networks:
      - network_api
    ports:
      - "8000:8000"
  test_authent:
    depends_on: 
      - fastapi
    image: image_test_python_fastapi
    container_name: test_authent_container
    environment:
      - LOG=1
      - TEST_FILE="test_authent.py"
    networks:
      - network_api
    volumes:
      - .:/usr/src/app
    command: tail -F anything
networks:
 network_api:
  driver: bridge
